### PR TITLE
feat(icons): add `letter-text` icon

### LINKED ISSUE #1676: Dropcap Icon
### Icon name

dropcap

### Use cases

Making a WYSIWYG that includes a dropcap button.

### Design ideas

image
image
image

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

### PR DESCRIPTION
**Pull request by GRA0007**
Closes #1676 

## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description

Adds a `letter-text` icon.

### Icon use case

- Inserting a drop cap into a rich text editor
- To indicate a letter in a paragraph of text

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [ ] The icons are solely my own creation.
- [x] The icons were originally created in #1676 by @jguddas 
- [ ] I've based them on the following Lucide icons: 
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Review by karsa-mistmere — APPROVED** [review]
LGTM!

File: icons/drop-cap.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15 12h6" />
+  <path d="M15 6h6" />
+  <path d="m3 13 4-8 4 8" />
```
Comment (jguddas) [inline icons/drop-cap.svg:14]:
```suggestion
  <path d="m3 13 3.553-7.724a.5.5 0 0 1 .894 0L11 13" 
```

**Comment by jguddas** [comment]
I think the name should be changed, `drop-cap` is not descriptive about what's inside the icon, but rather what it represents.

**Comment by GRA0007** [comment]
@jguddas what would you recommend? `letter-paragraph`?

**Comment by ericfennis** [comment]
@GRA0007 Or `letter-text`?

**Comment by GRA0007** [comment]
Yeah thanks @ericfennis, that sounds good

**Review by ericfennis — APPROVED** [review]

**Review by jguddas — APPROVED** [review]

**Pull request merged**